### PR TITLE
Allows retrieval of all local package contents via http://server/-/search/~

### PR DESCRIPTION
**Pull request by Verikon**

### CODE REVIEW

**Comment by Verikon** [comment]
If there's a preference for the wildcard I'm happy to change it upon request; also note the score attribute isn't required.

File: lib/search.js [hunk 1]
```diff
@@ -12,7 +12,9 @@ function Search() {
 }
 
 Search.prototype.query = function(q) {
-  return this.index.search(q)
+	return q === '~'
```
Comment (juanpicado) [inline lib/search.js]: I wonder why ~ and not a wildcard instead?

**Comment by juanpicado** [comment]
@Verikon  There isn't a way to "cheat" lunrjs to match all items?

File: lib/search.js (hunk 1 above)
Comment (Verikon) [inline lib/search.js]: Not entirely sure what you mean by a wildcard, an "*" (asterisk) ?

If so i can give you another commit with that, or whatever you would like.

**Comment by Verikon** [comment]
Querying lunr's index for all items makes a lot more sense and that was my initial approach. It's unclear how to produce a query which will return all items in Lunr's index ( https://lunrjs.com/docs/index.js.html ).

Baring that in mind, I instead made use of the storage.get method.

File: lib/search.js (hunk 1 above)
Comment (juanpicado) [inline lib/search.js]: Yes, exactly. That would be great. I think * is widely used for such things.

Comment (juanpicado) [inline lib/search.js]: @Verikon ping. I'd love to ship a release this week :)

Comment (Verikon) [inline lib/search.js]: @juanpicado doin it

**Comment by Verikon** [comment]
tested it, you can now list all local contents via an asterisk.

**Pull request merged**